src/pages/uxp/reference-js/Modules/FileSystem.md

```diff
@@ -165,7 +165,9 @@ await fs.close(fd);
 ## read(fd, buffer, offset, length, position, callback)
 Reads data in chunks from the file it refers to the file descriptor
 
+**Returns**: `Promise<Object>` - { bytesRead, buffer }  
```

Comment:
(not a new issue, just a moved line)

https://github.com/AdobeDocs/uxp/issues/86

But also, there is a bunch of information missing: What types do `bytesRead` and `buffer` have?

=> needs an interface for that object that describes it, and then it should be `Promise<InterfaceName>`.